### PR TITLE
Disable some unit tests for training

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

Talked with Jacky offline, these test cases are wrong. So we will disable the failed tests.

Onnxruntime-Linux-GPU is a new machine pool that will be used to replace "Linux-GPU-CUDA10" machine pool for all the Linux GPU CI build.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

**Review by alissonsol — APPROVED** [review]
Approve the changes to the two files.

File: onnxruntime/test/providers/cpu/model_tests.cc [hunk 1]
```diff
@@ -126,6 +126,40 @@ TEST_P(ModelTest, Run) {
...
+      {"softmax_cross_entropy_sum", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3_none_no_weight_negative_ignore_index_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none_weights", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_4d_log_prob", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_none", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1_mean_weight_negative_ignore_index", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_input_shape_is_NCd1d2d3d4d5_mean_weight", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_weight", "type error", {"onnx170"}},
+      {"softmax_cross_entropy_mean_no_weight_ignore_index_4d", "type error", {"onnx170"}},
 #endif
```
Comment (jcwchen) [inline onnxruntime/test/providers/cpu/model_tests.cc:163]: Just curious: how about those `test_sce_*` test failures in onnx181/?

Comment (snnn) [reply onnxruntime/test/providers/cpu/model_tests.cc:163]: I didn't see them failing.

Comment (snnn) [reply onnxruntime/test/providers/cpu/model_tests.cc:163]: I changed them to use the pre-generated data in ONNX 1.8.1 release, it works well.

**Review by jcwchen — APPROVED** [review]
LGTM. Thank you so much for the skip. I will let you know once ONNX has fixed backend data generation issue.

**Pull request merged**